dashboard/app/models/levels/poetry.rb

```diff
@@ -42,7 +42,11 @@ def shared_function_type
   end
 
   def self.standalone_app_names
-    [['Poetry', 'poetry'], ['Poetry HOC', 'poetry_hoc']]
+    [['Poetry', 'poetry'], ['Poetry HOC', 'poetry_hoc'],  ['Time Capsule', 'time_capsule']]
+  end
+
+  def self.subtypes_with_poems
+    %w(poetry_hoc time_capsule)
```

Comment:
Is there are a convention for when we use this shortcut to define an array vs the with square brackets like above in `standalone_app_names`?

Reply:
I just do what the linter tells me :) I think in this case because `standalone_app_names` is an array of arrays it is clearer to write it without this syntax.

Reply:
Sounds good! I haven't work as much with Ruby so the `%w` syntax was new to me. Thanks for explaining.